conf/application.conf

```diff
@@ -308,4 +308,8 @@ auditing {
       port = 8100
     }
   }
+}
+
+features {
+  welshVatEmailEnabled = false
```

Comment:
Think you have some of the reverted commit in this PR

Reply:
Whoopsie, will rebase and sort out now :)